src/compiler/sexp.rs

```diff
@@ -23,6 +28,70 @@ pub enum SExp {
     Atom(Srcloc, Vec<u8>),
 }
 
+pub fn random_atom_name<R: Rng + ?Sized>(rng: &mut R, min_size: usize) -> Vec<u8> {
```

Comment:
this looks like a function only used for tests. is it?
if so, it should have `#[cfg(test)]` above it.

also `random_sexp`, `random_atom` and `random_sexp`.

Reply:
Ya, i should get in that habit.

Reply:
Although i have built bins that use these as well in the full fuzz branch, so they'll come right back eventually.

Reply:
you can condition it to the fuzzing configuration as well.
I usually get warnings for unused functions if I don't mark up functions with the right configuration. Maybe because these are `pub` they become part of the external interface and always considered used.